DrugLabelExplorer/dle
# Ky/create search form

This PR implements the basic search view and search results view functionality. PR also includes basic Django config and skeleton to install the search app within the DLE project.

## Verification
* Start dev server with `python manage.py runserver`
* Navigate to "/" should bring you to the search view, searching text should bring you to the search results list page.
